src/components/contracts/utils/index.js

```diff
@@ -176,3 +176,17 @@ export const defaultContract = (fieldValues) => ({
   fieldValues,
   children: null,
 });
+
+export const previousContract = (previousContract) => {
+  return {
+    ...previousContract,
+    id: 0,
+  };
+};
+
+export const loadPreviousContract = (contracts) => {
```

Comment:
it's not "loading" and here it's more findLastContract(contracts)

Reply:
yes, I thought it would be better to pre fill the form with the data of the last contract. Maybe I should rename the function with findLastContract